regression/cbmc-java/coreModels/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+test.class
+--show-symbol-table
+^EXIT=0$
+^SIGNAL=0$
+^Symbol\s*\.*\: java\:\:org\.cprover\.CProver\.\<clinit\>\:\(\)V$
+--
+tests that the core models are  being loaded by checking if the static initializer for the CProver class was
```

Comment:
This is missing a second `--`, unless you really want the test to check that that text does not appear in the output.

Reply:
done